app/models/rbac.rb

```diff
@@ -96,6 +96,7 @@ def self.apply_rbac_to_associated_class?(klass)
   end
 
   def self.rbac_class(klass)
+    klass = klass.respond_to?(:klass) ? klass.klass : klass
```

Comment:
Yeah, this parameter should really be named `scope` or `scope_or_klass`, I think